### PR TITLE
docs: update cilium inline install

### PR DESCRIPTION
**Pull request by SixFive7**
# Pull Request

## What? (description)
Minor deduplication inside the Cilium install docs.

## Why? (reasoning)
Restores brevity of the document but keeps the previously missing prerequisite on method #4.

## Acceptance

Please use the following checklist:

- [x] you linked an issue (if applicable)
  https://taloscommunity.slack.com/archives/CMARMBC4E/p1646263069989849
- [ ] you included tests (if applicable)
- [ ] you ran conformance (`make conformance`)
- [ ] you formatted your code (`make fmt`)
- [ ] you linted your code (`make lint`)
- [x] you generated documentation (`make docs`)
- [ ] you ran unit-tests (`make unit-tests`)

---
This change is [Reviewable](https://reviewable.io/reviews/talos-systems/talos/5124)

### CODE REVIEW

File: website/content/docs/v1.0/Guides/deploying-cilium.md [hunk 1]
```diff
@@ -128,23 +128,8 @@ As such you should definitely not host this somewhere publicly accessible.
 
 ## Method 4: Helm manifests inline install
 
+This method requires that you have set your CNI to `none` as outlined [above](#machine-config-preparation).\
```
Comment (frezbo) [inline website/content/docs/v1.0/Guides/deploying-cilium.md:131]: @smira does gridsome render this?

Comment (smira) [reply website/content/docs/v1.0/Guides/deploying-cilium.md:131]: no idea really, but you can `cd website; gridsome develop` and verify it

Comment (smira) [reply website/content/docs/v1.0/Guides/deploying-cilium.md:131]: verified that it works

**Pull request merged**